app/lib/log_request_params.rb

```diff
@@ -1,6 +1,7 @@
 module LogRequestParams
   IGNORE_PARAMS = [
     /^authenticity_token$/,
```

Comment:
Replace with checking `filter_parameters` blacklist to filter query params.

Reply:
And add `token` to `https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/master/config/initializers/filter_parameter_logging.rb